src/client/src/ui/footer/model/index.js

```diff
@@ -1 +1 @@
-export { default as FooterModel } from './footerModel';
+export * from './footerModel';
```

Comment:
if we move the `FooterConnectionStatus` to another class this will have to change